src/Symfony/Component/Translation/Extractor/Visitor/AbstractVisitor.php

```diff
@@ -42,7 +42,7 @@ protected function addMessageToCatalogue(string $message, ?string $domain, int $
 
     protected function getStringArguments(Node\Expr\CallLike|Node\Attribute|Node\Expr\New_ $node, int|string $index, bool $indexIsRegex = false): array
     {
-        $args = $node instanceof Node\Expr\CallLike ? $node->getArgs() : $node->args;
+        $args = $node instanceof Node\Expr\CallLike ? $node->getRawArgs() : $node->args;
```

Comment:
```suggestion
```
Looks like `$args` variable is unnecessary before the `if (\is_string($index)) {` condition